ui/package.json

```diff
@@ -75,6 +76,7 @@
     "react-apollo": "^2.5.5",
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.8.6",
+    "react-event-listener": "^0.6.6",
```

Comment:
@helfi92 I had to install this package (dependency of `material-ui-pickers`) as I was running into `Can't resolve 'react-event-listener'` error when running the app. Ideally, we should migrate from `material-ui-pickers` to `@material-ui/pickers` v3.

Reply:
Agreed we should migrate. I think what's blocking us from migrating to the v3 version of the pickers package is https://github.com/taskcluster/taskcluster/issues/705.